awx/conf/settings.py

```diff
@@ -31,18 +31,18 @@
 # Store a special value to indicate when a setting is not set in the database.
 SETTING_CACHE_NOTSET = '___notset___'
 
-# Cannot store None in memcached; use a special value instead to indicate None.
+# Cannot store None in cache; use a special value instead to indicate None.
 # If the special value for None is the same as the "not set" value, then a value
 # of None will be equivalent to the setting not being set (and will raise an
 # AttributeError if there is no other default defined).
 # SETTING_CACHE_NONE = '___none___'
 SETTING_CACHE_NONE = SETTING_CACHE_NOTSET
```

Comment:
I'm betting that we don't _need_ to do this sort of thing anymore - it's likely that redis and its Python driver have support for storing `None`, `[]`, and `{}`.  That said, this could be a really hairy bit of code to tease apart, and I'm kind of scared to touch it since it's not really hurting anything keeping it this way.